### ISSUE TITLE
Forum Notifications

### ISSUE THREAD

**Issue by nesl247** [laravelio/portal#158]
Please add notifications for replies to the forums. It's annoying to have to go back and find your post and see if anyone has replied.

**Comment by ddimaria** [comment]
+1 on this as it's hard to follow up with any suggestions without going back.

**Comment by gamefanpage** [comment]
Hello, Any news about this features? will be great to have a notification system as a pop up/mail notification when an thread got a reply.

Thank you

**Comment by ddimaria** [comment]
@gamefanpage I don't think this package is being actively developed.  The last commit was 4 months ago, and the last release was about a year ago.  It stinks b/c this feature would really improve the user experience for new Laravel users who have questions...though there are many places to ask questions.

@driesvints you mentioned a 3.1 release.  Has any work been done on that?  If not, would you like one of us to create a PR to add this feature?   Thanks!

**Comment by driesvints** [comment]
@gamefanpage @ddimaria I'm actually working on a next release which is a totally new version, built from scratch. It'll be done when it's done :wink: 

Hopefully I can show a first preview in the next months.

**Comment by gamefanpage** [comment]
@driesvints  That is an awesome news :) Thank you for your hard work :)

**Comment by tdhsmith** [comment]
I would really appreciate this feature as well. As a newer contributor, it really discourages me from responding to too many questions because it's too much work to scan for updates.

At the very least, it would be nice to be able to access more than 5 of my recent posts so I don't have to bookmark them...

**Comment by driesvints** [comment]
@tdhsmith we're definitely trying to add this feature to the next version. At the very least that every subscriber to a topic gets a notification when a reply is posted.

**Comment by m1guelpf** [comment]
@driesvints Any updates on this?

**Comment by driesvints** [comment]
Unfortunately not. It's still on the todo list but I'm not sure if it'll make the upcoming beta.

**Comment by fetzi** [comment]
@driesvints should I send a PR for this? I would use database notifications. the thread author will be subscribed automatically and on reply you can select if you want to get notifications for that thread.

**Comment by driesvints** [comment]
@fetzi I'd like to take my time first to think this over a bit. I'll try to open an issue soon with a thorough description which we can discuss.

**Comment by fetzi** [comment]
Ok, perfect!

**Comment by driesvints** [comment]
I've written out two features for notifications, one is to subscribe to threads #270 , the other is user mentions #271. If anyone wants to pick them up, feel free to, otherwise I'll work on it whenever I find the time.
